ugs-platform/ugs-platform-plugin-joystick/src/main/java/com/willwinder/ugs/nbp/joystick/Settings.java

```diff
@@ -30,12 +31,31 @@ This file is part of Universal Gcode Sender (UGS).
  */
 public class Settings {
     public static final String SETTINGS_ACTIVE = "active";
+    public static final String SETTINGS_VERSION = "version";
 
     private static Preferences preferences = NbPreferences.forModule(JoystickService.class);
```

Comment:
Yes they are persisted. The downside is that the "Apply" button has little use as the settings are made instantly. But I figure that this can be added later when we have tried these features.